e2e/src/FiatConnectTransferOut.spec.js

```diff
@@ -0,0 +1,6 @@
+import { fiatConnectNonKycTransferOut } from './usecases/FiatConnectTransferOut'
+
+describe('FiatConnect Transfer Out', () => {
+  // deliberately not doing onboarding in beforeEach (since we sometimes want to re-use an account for returning user flow testing)
```

Comment:
```suggestion
  // deliberately not doing onboarding in beforeEach, since we'll want to re-use accounts, use fresh accounts, etc for these tests
```